client/src/testing/activated-route-stub.ts

```diff
@@ -13,7 +13,6 @@ export class ActivatedRouteStub {
   private subject = new ReplaySubject<ParamMap>(1);
 
   /** The mock paramMap observable */
-  // eslint-disable-next-line @typescript-eslint/member-ordering
```

Comment:
This failed as it didn't know about `@typescript-eslint/member-ordering` anymore, so I just removed it. Since `ng lint` passes, I assume it wasn't really necessary (at least not any more).